scipy/spatial/kdtree.py

```diff
@@ -199,6 +199,10 @@ class KDTree(object):
             >>> import sys
             >>> sys.setrecursionlimit(10000)
 
+    See Also
+    --------
+    cKDTree : Implementation of `KDTree` in Cython
```

Comment:
<a href="https://github.com/scipy/scipy/blob/master/scipy/spatial/ckdtree.pyx#L398">This</a> really looks like Cython to me.  For your statement to be correct, <u>every</u> line of code in that class definition must be pure C++, and I'm pretty certain `len` does not exist in the C++.